src/OpenTelemetry.Instrumentation.MassTransit/OpenTelemetry.Instrumentation.MassTransit.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+        <Description>OpenTelemetry instrumentation for MassTransit</Description>
+        <PackageTags>$(PackageTags);distributed-tracing;MassTransit</PackageTags>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="OpenTelemetry" Version="0.4.0-beta.2" />
```

Comment:
would it be enough to only reference `OpenTelemetry.Api`?

Reply:
It at least needs ListenerHandler and ActivitySource adapter which is not available in OpenTelemetry.Api package.